specification/maps/data-plane/Search/preview/2022-09-01-preview/examples/ReverseGeocoding.json

```diff
@@ -0,0 +1,86 @@
+{
+  "parameters": {
+    "api-version": "2022-09-01-preview",
+    "query": "-122.138681,47.630358",
+    "AzureKey": "[subscription-key]"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "geoJsonData": {
+          "type": "FeatureCollection",
+          "features": [
+            {
+              "type": "Feature",
+              "properties": {
+                "address": {
+                  "countryRegion": {
+                    "name": "United States"
+                  },
+                  "adminDistricts": [
+                    {
+                      "shortName": "WA"
+                    },
+                    {
+                      "shortName": "King County"
+                    }
+                  ],
+                  "formattedAddress": "15127 NE 24th St, Redmond, WA 98052",
+                  "locality": "Redmond",
```

Comment:
Do we want to have the neighborhood in the list?

Reply:
Yes. We have neighborhood information under address object, but it just not showing up in the response for this example.

Reply:
According to our discussion, neighborhood should be included in both forward and reverse geocoding API.